r/tests/testthat/test-dplyr-string-functions.R

```diff
@@ -866,3 +866,44 @@ test_that("str_like", {
     df
   )
 })
+
+test_that("str_pad", {
+  
```

Comment:
Should some of these tests use odd number `width`?

Reply:
Also: should some of these tests use `width` that's less than the number of characters in the input? The `stringr::str_pad()` docs show an example like that:
```r
# Longer strings are returned unchanged
str_pad("hadley", 3)
```

Reply:
Good point - have alternated between even and odd numbers for `width` now.

The second to last test uses a `width` value shorter than the input.